[2022-03-15 18:07:19,093] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: gcs_ingestion_dag_v02.download_dataset_task scheduled__2020-04-01T00:00:00+00:00 [queued]>
[2022-03-15 18:07:19,100] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: gcs_ingestion_dag_v02.download_dataset_task scheduled__2020-04-01T00:00:00+00:00 [queued]>
[2022-03-15 18:07:19,101] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 18:07:19,101] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-15 18:07:19,101] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 18:07:19,109] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-04-01 00:00:00+00:00
[2022-03-15 18:07:19,113] {standard_task_runner.py:52} INFO - Started process 752 to run task
[2022-03-15 18:07:19,115] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'gcs_ingestion_dag_v02', 'download_dataset_task', 'scheduled__2020-04-01T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/gcs_ingestion_dag_v02.py', '--cfg-path', '/tmp/tmpm05arlgz', '--error-file', '/tmp/tmpk92lk4vv']
[2022-03-15 18:07:19,116] {standard_task_runner.py:77} INFO - Job 24: Subtask download_dataset_task
[2022-03-15 18:07:19,150] {logging_mixin.py:109} INFO - Running <TaskInstance: gcs_ingestion_dag_v02.download_dataset_task scheduled__2020-04-01T00:00:00+00:00 [running]> on host 3570d6d4f2dc
[2022-03-15 18:07:19,185] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gcs_ingestion_dag_v02
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-04-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-04-01T00:00:00+00:00
[2022-03-15 18:07:19,185] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-15 18:07:19,186] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://www.dropbox.com/s/a1kl5e35j4o53mz/bandcamp-items-json.zip > /opt/***/bandcamp-items-json.zip']
[2022-03-15 18:07:19,194] {subprocess.py:85} INFO - Output:
[2022-03-15 18:07:34,274] {local_task_job.py:212} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2022-03-15 18:07:34,275] {process_utils.py:124} INFO - Sending Signals.SIGTERM to group 752. PIDs of all processes in the group: [753, 754, 752]
[2022-03-15 18:07:34,275] {process_utils.py:75} INFO - Sending the signal Signals.SIGTERM to group 752
[2022-03-15 18:07:34,275] {taskinstance.py:1408} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-03-15 18:07:34,275] {subprocess.py:99} INFO - Sending SIGTERM signal to process group
[2022-03-15 18:07:34,301] {process_utils.py:70} INFO - Process psutil.Process(pid=752, status='terminated', exitcode=0, started='18:07:18') (752) terminated with exit code 0
[2022-03-15 18:07:34,302] {process_utils.py:70} INFO - Process psutil.Process(pid=754, status='terminated', started='18:07:18') (754) terminated with exit code None
[2022-03-15 18:07:34,302] {process_utils.py:70} INFO - Process psutil.Process(pid=753, status='terminated', started='18:07:18') (753) terminated with exit code None
